src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php

```diff
@@ -60,7 +60,7 @@ public function loadTypeGuesser(): ?FormTypeGuesserInterface
     protected function loadTypeExtensions(): array
     {
         return [
-            new Type\FormTypeValidatorExtension($this->validator, $this->legacyErrorMessages, $this->formRenderer, $this->translator),
+            new Type\FormTypeValidatorExtension($this->validator, $this->formRenderer, $this->translator),
```

Comment:
I think I must do same deprecation layer for this current class `ValidatorExtension`, are you agree ?

I didn't find usage of `ValidatorExtension` (except in tests), do you know how it used ?

Reply:
indeed. Maybe when the component is used standalone? Anyway, that's not our call.

Reply:
Those extensions are indeed meant for standalone usage of the component. FrameworkBundle registers everything through the DependencyInjectionExtension (which allows lazy-loading things)